components/esp_lvgl_port/include/esp_lvgl_port.h

```diff
@@ -228,6 +246,39 @@ lv_indev_t *lvgl_port_add_navigation_buttons(const lvgl_port_nav_btns_cfg_t *but
 esp_err_t lvgl_port_remove_navigation_buttons(lv_indev_t *buttons);
 #endif
 
+
+#if __has_include ("usb/hid_host.h")
```

Comment:
I see that you are following an already established pattern, just a note about it: it seems that this #if prevents Doxygen from seeing the declaration of the function, so it's not going to be visible in the API reference, once we have it.